xml/System.Security.Cryptography/AesCcm.xml

```diff
@@ -172,13 +196,24 @@
         <Parameter Name="associatedData" Type="System.Byte[]" />
       </Parameters>
       <Docs>
-        <param name="nonce">To be added.</param>
-        <param name="plaintext">To be added.</param>
-        <param name="ciphertext">To be added.</param>
-        <param name="tag">To be added.</param>
-        <param name="associatedData">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="nonce">The nonce associated with this message, which should be a unique value for every operation with the same key.</param>
+        <param name="plaintext">The content to encrypt.</param>
+        <param name="tag">The byte array to receive the genenerated authentication tag.</param>
```

Comment:
```suggestion
        <param name="tag">The byte array to receive the generated authentication tag.</param>
```

Reply:
Sheesh, you make one typo... and then copy/paste it 3 more times... :smile:.